v22.1/map-sql-activity-to-app.md

```diff
@@ -0,0 +1,57 @@
+~~~
+
+## Filter by application name
+
+Once you set the application name, the [DB Console](ui-overview.html) lets you [filter database workloads by application name](ui-statements-page.html#filter).
+
+<img src="{{ 'images/v22.1/movr-app.png' | relative_url }}" alt="Movr app filter" style="border:1px solid #eee;max-width:80%" />
+
+If parts of your applications or known microservices are experiencing performance degradation, you can filter for the database workload tracing back to that part of your application directly in the DB Console. You can quickly identify whether there were database performance problems and if so, troubleshoot the issue using native [SQL observability touch points](#trace-sql-activity-using-metrics) in the DB Console.
+
+You can also programmatically filter `crdb_internal` tables [`crdb_internal.statement_statistics`](crdb-internal.html#statement_statistics) and [`crdb_internal.transaction_statistics`](crdb-internal.html#transaction_statistics) by application name. This example shows the first 60 characters of query text and statement statistics for queries associated with the `movr_app` application:
+
+{% include copy-clipboard.html %}
+~~~sql
+SELECT app_name, substring(metadata ->> 'query',1,60) AS statement_text,
+   metadata -> 'distsql' AS is_distsql,
+   metadata -> 'fullScan' AS has_full_scan,
+   metadata -> 'vec' AS used_vec,
+   statistics -> 'execution_statistics' -> 'contentionTime' -> 'mean' AS contention_time_mean,
+   statistics -> 'statistics' -> 'cnt' AS execution_count,
+   statistics -> 'statistics' -> 'firstAttemptCnt' AS num_first_attempts,
+   statistics -> 'statistics' -> 'numRows' -> 'mean' AS num_rows_returned_mean,
+   statistics -> 'statistics' -> 'rowsRead' -> 'mean' AS num_rows_read_mean,
+   statistics -> 'statistics' -> 'runLat' -> 'mean' AS runtime_latency_mean
+FROM movr.crdb_internal.statement_statistics
+WHERE app_name = 'movr_app';
+~~~
+
+~~~
+  app_name |                        statement_text                        | is_distsql | has_full_scan | used_vec | contention_time_mean | execution_count | num_first_attempts | num_rows_returned_mean | num_rows_read_mean | runtime_latency_mean
+-----------+--------------------------------------------------------------+------------+---------------+----------+----------------------+-----------------+--------------------+------------------------+--------------------+------------------------
+  movr_app | SELECT * FROM users                                          | true       | true          | true     |                    0 |               1 |                  1 | 5E+1                   | 5E+1               |              0.002202
+  movr_app | SELECT name, count(rides.id) AS sum FROM users JOIN rides ON | true       | true          | true     |                    0 |               1 |                  1 | 1E+1                   | 5.5E+2             |              0.005338
+...
+~~~
+
+## Trace SQL activity using metrics
```

Comment:
Nothing to change in this section but calling out that once we have a global time picker, we should show users and call out that the time picker is shared across the time-series metrics dashboards with SQL Activity information. This would be a great place for this content.

Reply:
What's the Jira for the global time picker?

Reply:
Here's the JIRA: https://cockroachlabs.atlassian.net/browse/CRDB-13581